### DISCUSSION TITLE
Usage guide or Tips and Tricks

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#457] [Ideas]
As we experienced multiple answered questions and issues at least twice(redundant), maybe we could agree on making a "USAGE " page for the README? And what should this consist of or what should be the structure of the page? Can you suggest a workflow? And also maybe someone there is willing to write it, I don't know how. Haha

Examples are how to adjust the waybar, set wallpaper, etc.

**Comment by zer0cha0s** [comment]
Tittu has already started a wiki for installation and theming, it may be worth putting an FAQ in there.

**Reply by prasanthrangan** [reply]
I agree, the wiki and readme is kinda outdated... It needs to be updated 
Will add new pages for FAQ's and Features (how to use guide)

**Comment by kRHYME7** [comment]
We badly needed How to Guide Sooner. Thanks to Ksk royal for creating a [video](https://www.youtube.com/watch?v=WuZ2T6D_9yI) on this. While this might cause a lot of issue reports, it's also attracting new contributors, so it's still a win overall. A big shout out to @prasanthrangan and all contributor for all their hard work!

As a result of this, we're likely to encounter issues that are redundant, have already been closed a long time ago, or are essentially "how-to" questions.

I'm still learning how to express my thoughts effectively, but I'm giving it my best shot. I'm also not yet super proficient at creating "How to's".

To tackle this, I'm working on creating a more user-friendly CLI for Hyprdots. This CLI will support uncommon or one-time setup tasks, such as theme patching with sddmwall.sh, and more. This will make the instructions less intimidating at first glance, especially if someone is planning to create a Guide soon.

Here's an example of how the CLI might look:
```
Usage: Hyprdots [Options] [] [Parameters]
   restore
   install
   theme [set|next|prev|select|patch]
   wallpaper [set|next|prev|select]
   sddm [set]
   wallbash [toggle]
```
This CLI requires the following: export PATH=$HOME/.local/bin:$PATH (this is already implemented in zshrc).

I'm currently working on this in a gist, which you can find [here](https://gist.github.com/kRHYME7/8c64f94fcbf7d123bdc95ab10a85d2fb). This script uses $@ to call the function and handles anything inside the function only, ensuring better isolation for each function.

**Comment by T-Crypt** [comment]
@kRHYME7 

If you want to help me, I'm working on the FAQ and Common issues pages in wiki right now. You could help me by making markdown files for the common issues!

https://github.com/prasanthrangan/hyprdots/wiki/FAQ

I'm out most of this week with my family for the holidays but Ill finish up next week!

**Reply by kRHYME7** [reply]
I'll do my best

**Reply by kRHYME7** [reply]
Very very very not yet complete.
[List of Qs and Issues](https://gist.github.com/kRHYME7/b31bcfe3358c043113e1c604644a4bd2)
